arduino/smartcar/smartcar.ino

```diff
@@ -19,14 +19,15 @@ void setup() {
 }
 
 void loop() {
-  Serial.println(front.getDistance());
+  int currDist = front.getDistance();
```

Comment:
Good idea to call front.getDistance() once and save it once per iteration instead of calling it on every if statement